packages/fx-core/src/plugins/solution/fx-solution/v2/getQuestions.ts

```diff
@@ -289,13 +292,27 @@ export async function getQuestions(
     } else {
       plugins = getAllV2ResourcePlugins();
     }
-    plugins = plugins.filter((plugin) => !!plugin.deploy && plugin.displayName !== "AAD");
-    if (plugins.length === 0 && inputs.skipAadDeploy !== "no") {
+
+    let filterAadPlugin = true;
+    if (
+      isAadManifestEnabled() &&
+      (inputs.platform === Platform.CLI_HELP || inputs.platform === Platform.CLI)
+    ) {
+      filterAadPlugin = false;
+    }
+
+    if (filterAadPlugin) {
```

Comment:
Perhaps it's better to merge this if block with the if block on Line 297

Reply:
OK, thanks